tools/isobuild/linker.js

```diff
@@ -612,6 +612,17 @@ var getHeader = function (options) {
     chunks.push("var " + options.packageVariables.join(', ') +
                 ";\n\n");
   }
+  if (options.combinedServePath || options.name) {
+    chunks.push("/* Package reflection */\n");
+    if (options.name) {
+      chunks.push("var PackageName = " + JSON.stringify(options.name) + ";\n");
+    }
+    if (options.combinedServePath) {
+      chunks.push("var PackagePath = " +
```

Comment:
In my world this is the same as the path in the manifest file. So you can use it to resolve the entry in the manifest file where you can get some extra information.